ES/ESbgy004.xml

```diff
@@ -735,8 +739,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <desc>For the same note, cp. 
                and<ref type="mss" corresp="ESath007">ATH-007</ref>, <locus target="#93ra">93ra</locus>. 
                In <ref type="mss" corresp="ESbgy004">BGY-004</ref>, as in<ref type="mss" corresp="ESath007">ATH-007</ref>, the word እምዐረቢ or ዐረቢ (“[from] Arabic”) is  omitted.</desc>
-         </relation>                                 
-            <relation name="skos:closeMatch" active="ESbgy004#GHsub2" passive="ESgbi002#GHsub2"><desc>the same note in <ref type="mss" corresp="ESgbi002">GBI-002</ref>, see also the edition provided in <listBibl type="secondary"><bibl><ptr target="bm:Pisani2019GH"/></bibl></listBibl></desc></relation>
+         </relation>          
+            
+            <relation name="skos:closeMatch" active="ESbgy004#subGH1" passive="ESmm002#subGH1">
```

Comment:
This seems to me rather to be a colophon of the translation that could be encoded as a textpart in the work record?

Reply:
see https://github.com/BetaMasaheft/Works/pull/14